### PR TITLE
[DOC] Apply Domain explained + Preprocessing for Unsupervised

### LINKED ISSUE #5805: widget (apply domain) bug with PCA
**What's wrong?**
cannot transform data as expecting.
image

image
the transformed data is wrong.

### **the following image represented the expecting correct transformed data.**

image

**How can we reproduce the problem?** 

[superpixel_0124.zip](https://github.com/biolab/orange3/files/7940610/superpixel_0124.zip)

**What's your environment?**

- Operating system: linux
- Orange version:3.31.1
- How you installed Orange: conda

### PR DESCRIPTION
**Pull request by ajdapretnar**
##### Issue

Resolves #5805.
Unsupervised widgets had no information on preprocessing.

##### Description of changes
Explain why template data is needed for transformation in Apply Domain.
Add info on preprocessing in unsupervised widgets.

##### Includes
- [ ] Code changes
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: doc/visual-programming/source/widgets/data/applydomain.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ Given dataset and template transforms the dataset.
 
 The widget receives a dataset and a template dataset used to transform the dataset.
 
+Side note
+--------
```
Comment (markotoplak) [inline doc/visual-programming/source/widgets/data/applydomain.md:22]: Looks good, clean and simple.

File: doc/visual-programming/source/widgets/unsupervised/mds.md [hunk 2]
```diff
@@ -57,6 +57,16 @@ widget. It is in many respects similar to the `Scatter Plot
 <../visualize/scatterplot>` widget, so we recommend reading that
 widget's description as well.
 
+Preprocessing
+-------------
+
+MDS uses default preprocessing if necessary. It executes it in the following order:
```
Comment (markotoplak) [inline doc/visual-programming/source/widgets/unsupervised/mds.md:63]: Doesn't MDS only apply these preprocessors if the data was given directly, without going through distances? It that sis so, we should mention this.

Comment (ajdapretnar) [reply doc/visual-programming/source/widgets/unsupervised/mds.md:63]: You're right, I will add it.

**Comment by markotoplak** [comment]
Thanks!

**Pull request merged**